CHANGELOG.md

```diff
@@ -37,6 +38,7 @@ These are the section headers that we use:
 - Update `FeedbackDataset.__init__` to allow passing a vector_settings list. ([#4055](https://github.com/argilla-io/argilla/pull/4055))
 - Update `FeedbackDataset.push_to_argilla` to also push vector settings. ([#4055](https://github.com/argilla-io/argilla/pull/4055))
 - Update `FeedbackDatasetRecord` to allow to create records with vectors. ([#4043](https://github.com/argilla-io/argilla/pull/4043))
+- Update `POST /api/v1/me/datasets/:dataset_id/records/search` endpoint to allow to search records with vectors. ([#4019](https://github.com/argilla-io/argilla/pull/4019))
 
```

Comment:
We should add the breaking change here somehow about using ElasticSearch by default.